### PR TITLE
Merge pull request #78601 from foxit64/master

### PR DESCRIPTION
**Pull request by foxit64**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions (Debian and Arch-Linux)
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Frostman** [comment]
Tested on my system (x86_64) - builds fine and binary works correctly. Diff LGTM as well.

File: pkgs/servers/monitoring/telegraf/default.nix [hunk 1]
```diff
@@ -27,6 +27,6 @@ buildGoPackage rec {
     description = "The plugin-driven server agent for collecting & reporting metrics.";
     license = licenses.mit;
     homepage = https://www.influxdata.com/time-series-platform/telegraf/;
-    maintainers = with maintainers; [ mic92 roblabla ];
+    maintainers = with maintainers; [ mic92 roblabla foxit64 ];
```
Comment (Mic92) [inline pkgs/servers/monitoring/telegraf/default.nix]: also add yourself to maintainers.nix

**Comment by Mic92** [comment]
@GrahamcOfBorg test telegraf

File: pkgs/servers/monitoring/telegraf/default.nix (hunk 1 above)
Comment (foxit64) [reply pkgs/servers/monitoring/telegraf/default.nix]: Done! https://github.com/NixOS/nixpkgs/pull/78709

**Pull request merged**